code/__DEFINES/layers.dm

```diff
@@ -201,6 +201,8 @@
 
 #define ADMIN_POPUP_LAYER 1
 
+//Layer for screentips
```

Comment:
if you put three ///s, it will auto-document the proc on our end!

Reply:
Done! Thanks!